implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -322,6 +373,29 @@ impl PartialEq for IdentityConfig {
 
 impl Eq for IdentityConfig {}
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct EnrollmentStatus {
+    pub is_enrolled: bool,
+    pub created_at: SystemTime,
+}
+
+impl Display for EnrollmentStatus {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        if self.is_enrolled {
+            writeln!(f, "Enrolled: yes")?;
+        } else {
+            writeln!(f, "Enrolled: no")?;
+        }
+
+        match self.created_at.duration_since(UNIX_EPOCH) {
+            Ok(time) => writeln!(f, "Timestamp: {}", time.as_secs())?,
```

Comment:
I think it would be best to show a human-readable format here. I'd suggest adding the [`time`](https://github.com/time-rs/time) dependency and format the `SystemTime` to [Iso8601](https://docs.rs/time/latest/time/format_description/well_known/struct.Iso8601.html#)

Reply:
@adrianbenavides
Please let me know your thoughts on how to handle that point.

Reply:
OK, let's not worry about this. I'll open a new issue to tackle this in a separate PR.